src/apps/answers/tests/test_answers.py

```diff
@@ -3508,7 +3508,7 @@ async def test_validat_multiinformant_assessment_fail_not_manager(self, client,
 
         response = await client.get(url)
 
-        assert response.status_code == http.HTTPStatus.FORBIDDEN
+        assert response.status_code == http.HTTPStatus.BAD_REQUEST
```

Comment:
Permissions-related errors should remain `FORBIDDEN`. (I guess this will be reverted automatically once you make the other requested changes 😅)

Reply:
I think there was an issue in how the test was structure and I need to applied some shanges but the test now expects FORBIDDEN and pass